src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font for 'Roboto' to ensure consistent rendering across different systems. For example: `font-family: 'Roboto', Arial, sans-serif;`